clients/src/test/java/org/apache/kafka/common/serialization/SerializationTest.java

```diff
@@ -63,196 +87,59 @@ public void testSerdeFromNotNull() {
     }
 
     @Test
-    public void testStringSerializer() {
+    public void testStringWithEncodingSerde() {
```

Comment:
Nit: Again, we should find a more descriptive name.  Perhaps: `stringSerdeShouldSupportDifferentEncodings()`